src/map/lua/lua_baseentity.cpp

```diff
@@ -9819,6 +9822,39 @@ sol::table CLuaBaseEntity::getNotorietyList()
     return table;
 }
 
+/************************************************************************
+ *  Function: setClaimable(...)
+ *  Purpose : sets m_IsClaimable for a mob
+ *  Example : mob:setClaimable(false)
+ *  Notes   :
+ ************************************************************************/
+
+void CLuaBaseEntity::setClaimable(bool claimable)
+{
+    XI_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_MOB);
```

Comment:
Do we want to continue to use breaks in these bindings, or return gracefully and log?

Reply:
Yeah, agreed